docker-compose.yml

```diff
@@ -124,6 +124,18 @@ services:
     volumes:
       - ./pgadmin_servers.json:/pgadmin4/servers.json
 
+  grafana:
+    container_name: koku_grafana
+    build:
+        context: grafana
+        dockerfile: Dockerfile-grafana
+    ports:
+        - 3000:3000
```

Comment:
Could we change port number?

The sources-api uses port 3000, so I can envision some issues if we had sources-api set up locally.

Reply:
Ah.